administrator/language/de-DE/com_guidedtours.ini

```
@@ -95,7 +95,9 @@ COM_GUIDEDTOURS_TYPE_REDIRECT_URL_DESC="Relative URL der Seite, zu welcher der S
 COM_GUIDEDTOURS_TYPE_REDIRECT_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_LABEL="Relative URL"
 COM_GUIDEDTOURS_URL_DESC="Relative URL der Seite, von der aus die Tour starten soll, z.&#8239;B. administrator/index.php?option=com_guidedtours&view=tours für die Listenansicht der Touren."
+COM_GUIDEDTOURS_USERSTATE_BADTOURID="Die Tour-ID muss positiv sein."
```

```suggestion
COM_GUIDEDTOURS_USERSTATE_BADTOURID="Die Tour-ID darf nicht negativ sein."
```
Hmm da gefällt mir schon der originale string nicht. Da wir hier aber auch 1:1 übersetzen kann ich auch mit "positiv" leben wollte es nur ansprechen wie du dazu stehst.